BuildAndTest.proj

```diff
@@ -75,13 +76,17 @@
 
       <!-- MakeConst unit tests tracked by https://github.com/dotnet/roslyn/issues/5918 -->
       <TestAssemblies Condition="'$(Test64)' != 'true'" 
-                      Include="$(OutputDirectory)\**\$(IncludePattern)" />
+                      Include="$(OutputDirectory)\$(IncludePattern)" />
```

Comment:
@davkean The idea behind the DeployDesktopTestRuntime project is that it isolates a specific set of tests and test dependencies to a particular directory. If other tests are deploying to a different directory I would consider that a bug.